contracts/test/integration/IntegrationTestOracleUpgrade.t.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {ERC20, IERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+
+import {MockPCVDepositV2} from "../../mock/MockPCVDepositV2.sol";
+import {IPCVDeposit} from "../../pcv/IPCVDeposit.sol";
+import {MockERC20} from "../../mock/MockERC20.sol";
+import {OraclePassThrough} from "../../oracle/OraclePassThrough.sol";
+import {ICore} from "../../core/ICore.sol";
+import {Core} from "../../core/Core.sol";
+import {IVolt, Volt} from "../../volt/Volt.sol";
+import {PriceBoundPSM, PegStabilityModule} from "../../peg/PriceBoundPSM.sol";
+import {getCore, getMainnetAddresses, VoltTestAddresses} from "../unit/utils/Fixtures.sol";
+import {ERC20CompoundPCVDeposit} from "../../pcv/compound/ERC20CompoundPCVDeposit.sol";
+import {Vm} from "./../unit/utils/Vm.sol";
+import {DSTest} from "./../unit/utils/DSTest.sol";
+import {MainnetAddresses} from "./fixtures/MainnetAddresses.sol";
+
+import {Constants} from "../../Constants.sol";
+
+import "hardhat/console.sol";
+
+contract IntegrationTestOracleUpgrade is DSTest {
+    using SafeCast for *;
+    PriceBoundPSM private psm;
+    ICore private core = ICore(MainnetAddresses.CORE);
+    ICore private feiCore = ICore(MainnetAddresses.FEI_CORE);
+    IVolt private volt = IVolt(MainnetAddresses.VOLT);
+    IVolt private fei = IVolt(MainnetAddresses.FEI);
+    IVolt private underlyingToken = fei;
+
+    /// ------------ Minting and RateLimited System Params ------------
+
+    uint256 public constant mintAmount = 10_000_000e18;
+    uint256 public constant bufferCap = 10_000_000e18;
+    uint256 public constant individualMaxBufferCap = 5_000_000e18;
+    uint256 public constant rps = 10_000e18;
```

Comment:
we don't need these for this test do we, the minting params & the fei references

Reply:
removed